week-2/mandatory/2-ecommerce-db/task.md

```diff
@@ -7,8 +7,79 @@ In this homework, you are going to work with an ecommerce database. In this data
 Below you will find a set of tasks for you to complete to set up a database for an e-commerce app.
 
 To submit this homework write the correct commands for each question here:
+
 ```sql
 
+1. SELECT name, address FROM customers WHERE country='United States';
+
+2. SELECT * FROM customers ORDER BY name ASC;  -- it already orders asc by default --
+
+3. SELECT * FROM products WHERE product_name LIKE '%socks%';
+
+4. SELECT products.id, products.product_name, pa.unit_price, pa.supp_id
+    FROM products JOIN product_availability pa ON products.id=pa.prod_id
+    WHERE pa.unit_price > 100;
+
+5. SELECT products.product_name most_expensive_products, pa.unit_price
+    FROM products JOIN product_availability pa ON products.id=pa.prod_id
+    ORDER BY pa.unit_price DESC
+    LIMIT 5;
+
+6. SELECT products.product_name, pa.unit_price, sup.supplier_name FROM
+    products JOIN product_availability pa ON
+    (products.id=pa.prod_id) JOIN suppliers sup ON
+    (pa.supp_id=sup.id);
+
+7. SELECT p.product_name, sup.supplier_name FROM
+    products p JOIN product_availability pa ON
+    (p.id=pa.prod_id) JOIN suppliers sup ON
+    (pa.supp_id=sup.id) WHERE sup.country='United Kingdom';
+
+8. SELECT cust.id cust_id, orders.id order_id, orders.order_reference,
+    orders.order_date, oi.quantity * pa.unit_price as total_cost
+    FROM customers cust JOIN orders ON (cust.id=orders.customer_id)
+    JOIN order_items oi ON (orders.id=oi.order_id)
+    JOIN product_availability pa ON (oi.supplier_id=pa.supp_id)
+    WHERE cust.id=1;
```

Comment:
The question is a bit not clear here, but it was meant to ask for orders and their total price (so grouping and summarising the order items for each order)
so if order A has 5 items, the total will be the sum of (quantity of each order * unit price)